tests/managed_services/mas_debezium/conftest.py

```diff
@@ -0,0 +1,210 @@
+        kafka_status_samples = TimeoutSampler(
+            wait_timeout=KAFKA_TIMEOUT,
+            sleep=10,
+            func=lambda: kafka_mgmt_api_instance.get_kafka_by_id(
+                id=kafka_instance.id
+            ).status
+            == "ready",
+        )
+        for sample in kafka_status_samples:
+            if sample:
+                break
+        kafka_ready_api = kafka_mgmt_api_instance.get_kafka_by_id(id=kafka_instance.id)
+        LOGGER.info(f"Kafka instance is ready:\n{kafka_ready_api}")
+        yield kafka_ready_api
+    except TimeoutExpiredError:
+        LOGGER.error(
+            f"Timeout expired. Kafka creation status: "
+            f"{kafka_mgmt_api_instance.get_kafka_by_id(id=kafka_instance.id).status}"
+        )
+        raise
+
+
+@pytest.fixture(scope="class")
+def kafka_instance_client(kafka_instance_ready, access_token):
+    # https://github.com/redhat-developer/app-services-sdk-python/tree/main/sdks/kafka_instance_sdk
+    configuration = rhoas_kafka_instance_sdk.Configuration(
+        host=kafka_instance_ready.admin_api_server_url, access_token=access_token
+    )
+
+    with rhoas_kafka_instance_sdk.ApiClient(
+        configuration=configuration
+    ) as kafka_api_client:
+        yield kafka_api_client
+
+
+@pytest.fixture(scope="class")
+def kafka_instance_sa(kafka_instance_client, service_accounts_api_instance):
+    service_account_create_request_data = ServiceAccountCreateRequestData(
+        name=KAFKA_SA_NAME,
+        description=f"{KAFKA_NAME} instance service-account",
+    )
+    kafka_sa = service_accounts_api_instance.create_service_account(
+        service_account_create_request_data=service_account_create_request_data
+    )
+    assert (
+        kafka_sa.id
+    ), f"Failed to create service-account for kafka. API response:\n{kafka_sa}"
+    LOGGER.info(f"kafka service-account:\n{kafka_sa}")
+
+    yield kafka_sa
+
+    service_accounts_api_instance.delete_service_account(id=kafka_sa.id, async_req=True)
+
+
+@pytest.fixture(scope="class")
+def kafka_sa_acl(kafka_instance_client, kafka_instance_sa):
+    # Binding the service-account instance to kafka with privileges
+    # via AclBinding instance
+    acl_api_instance = acls_api.AclsApi(api_client=kafka_instance_client)
+
+    # acl binding attributes
+    resources = ["GROUP", "TOPIC", "TRANSACTIONAL_ID"]
+    resource_name = "*"
+    pattern_type = AclPatternType("LITERAL")
+    permission = AclPermissionType("ALLOW")
+    principal = f"User:{kafka_instance_sa.id}"
+    operation = AclOperation("ALL")
+
+    for resource in resources:
+        resource_type = AclResourceType(resource)
+        acl_binding = AclBinding(
+            resource_type=resource_type,
+            resource_name=resource_name,
+            pattern_type=pattern_type,
+            permission=permission,
+            principal=principal,
+            operation=operation,
+        )
+        acl_api_instance.create_acl(acl_binding=acl_binding)
```

Comment:
@rnetser acls are being cleaned up along with the kafka instance they are bound to, similar to topics

Reply:
@myakove it seems as if the API should throw an exception for [unsuccessful attempt](https://github.com/redhat-developer/app-services-sdk-python/blob/main/sdks/kafka_instance_sdk/docs/AclsApi.md#create_acl:~:text=(acl_binding)-,except%20rhoas_kafka_instance_sdk.ApiException%20as%20e%3A,-print(%22Exception)

Anyway in case of a missing acl binding the consumer pod will fail to consume from the tested topic due missing privilege, therefore I've removed the code that asserts the acls binding.